app/main.py

```diff
@@ -1,5 +1,61 @@
-class Cargo:
-    def __init__(self, weight: int) -> None:
-        self.weight = weight
+from typing import List, Optional
+
+
+class BaseRobot:
+    def __init__(self, name: str, weight: float,
+                 coords: Optional[List[int]] = None) -> None:
+        self.name: str = name
+        self.weight: float = weight
+        self.coords: List[int] = coords if coords is not None else [0, 0]
+
+    def go_forward(self, step: int = 1) -> None:
+        self.coords[1] += step
+
+    def go_back(self, step: int = 1) -> None:
+        self.coords[1] -= step
+
+    def go_right(self, step: int = 1) -> None:
+        self.coords[0] += step
+
+    def go_left(self, step: int = 1) -> None:
+        self.coords[0] -= step
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
 
-# write your code here
+class FlyingRobot(BaseRobot):
+    def __init__(self, name: str, weight: float,
+                 coords: Optional[List[int]] = None) -> None:
+        if coords is None:
+            coords = [0, 0, 0]
+        else:
+            coords += [0] * (3 - len(coords))
```

Comment:
The same comment as above
```suggestion
        coords = coords or [0, 0, 0]
```